deebot_client/commands/xml/__init__.py

```diff
@@ -25,22 +25,25 @@
     "GetBatteryInfo",
     "GetChargeState",
     "GetCleanLogs",
+    "GetCleanSpeed",
     "GetCleanSum",
     "GetError",
-    "GetFanSpeed",
     "GetLifeSpan",
     "GetPos",
     "PlaySound",
+    "SetCleanSpeed",
 ]
 
 # fmt: off
 # ordered by file asc
 _COMMANDS: list[type[XmlCommand]] = [
     GetBatteryInfo,
     GetCleanLogs,
+    GetCleanSpeed,
     GetError,
     GetLifeSpan,
     PlaySound,
+    SetCleanSpeed
```

Comment:
Should be file-based sorted and not alphanumeric

Reply:
Done